best_practices/security.rst

```diff
@@ -43,14 +43,14 @@ which uses a login form to load users from the database:
 
 .. code-block:: yaml
 
-    # app/config/security.yml
+    # config/packages/security.yaml
     security:
         encoders:
-            AppBundle\Entity\User: bcrypt
+            App\Entity\User: bcrypt
 
         providers:
             database_users:
-                entity: { class: AppBundle:User, property: username }
+                entity: { class: App:User, property: username }
```

Comment:
This could be `App\Entity\User` ... but it may be better to do that change in all docs in a separate PR.

Reply:
Yea, I agree. The namespace is now small enough that this slightly longer format is worth it because it's more clear.

Reply:
I've actually just made this change, as this is the only spot in the docs where this appears (there are still many references of `AppBundle:`, but this is the only one that is `App:`). As we migrate away from `AppBundle`, we can also change to not use the shortcut.